pkg/build/types/parser/deffile_test.go

```diff
@@ -215,7 +217,8 @@ func TestParseDefinitionFile(t *testing.T) {
 			}
 
 			if !reflect.DeepEqual(defTest, defCorrect) {
-				t.Fatal("parsed definition did not match reference")
+				b, _ := json.Marshal(defTest)
+				t.Fatalf("parsed definition did not match reference, %s", string(b))
```

Comment:
Perhaps consider using `(*testing.T).Log()` for `b`, to make it easier to read the actual failure?

Reply:
Ahh - yeah, this was a quick debug change.... I will tidy it it up though rather than remove it as it is useful when working in this area.